src/components/navigation/www/navigation.jsx

```diff
@@ -347,7 +349,8 @@ var Navigation = React.createClass({
 var mapStateToProps = function (state) {
     return {
         session: state.session,
-        permissions: state.permissions
+        permissions: state.permissions,
+        searchTerm: state.searchTerm
```

Comment:
@TheGrits whoops I missed one thing – because the reducer is called `navigation`, the value will end up being stored in `state.navigation`. So, I think we need to change this to `searchTerm: state.navigation`